app/Services/PasswordService.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Services;
+
+use Illuminate\Support\Facades\Hash;
+use App\User;
+
+class PasswordService 
+{
+	public function edit($id)
+	{
+		$user = User::findOrFail($id);
+
+		return $user;
+	}
+
+	public function changePassword($data, $id)
+	{
+		try {
+			$user = User::findOrFail($id);
+			$current_password = $user->password;
+
+			if (Hash::check($data['old_password'], $current_password)) {
+				$new_password = Hash::make($data['old_password']);
+				$user->password = $new_password;
```

Comment:
```suggestion
				$user->password = Hash::make($data['old_password']);
```
$newPassword sài có 1 lần ko cần define biến đâu nhé